x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/privileges/11_builtin.yml

```diff
@@ -15,5 +15,5 @@ setup:
   # This is fragile - it needs to be updated every time we add a new cluster/index privilege
   # I would much prefer we could just check that specific entries are in the array, but we don't have
   # an assertion for that
-  - length: { "cluster" : 43 }
+  - length: { "cluster" : 44 }
```

Comment:
YAML test does not support checking feature flag. There is also no good way to check individual privilege. So I chose to skip this test for non-snapshot build (e.g. release test) in the gradle setup.